targets/TARGET_Realtek/TARGET_AMEBA/RTWInterface.cpp

```diff
@@ -19,12 +19,15 @@
 
 #include "RTWInterface.h"
 #include "mbed_interface.h"
-#include "rtw_emac.h"
 
+#include "Rtw_emac.h"
+#include "EMAC.h"
 #include "wifi_constants.h"
 #include "wifi_conf.h"
-#include "lwip_stack.h"
 
+#include "LWIPStack.h"
```

Comment:
You shouldn't be including `LWIPStack.h`

Reply:
Thanks. Confirmed to delete LWIPStack.h .